Description: Drop and relax dependencies
 Dropped
 - rustainers: Dev dependency, not in Debian.

 Relaxed
 - base64: Dependency, only newer version in Debian.
Author: Tobias Deiminger <tobias.deiminger@posteo.de>
Last-Update: 2025-09-08
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: nethsm-sdk-rs/Cargo.toml
===================================================================
--- nethsm-sdk-rs.orig/Cargo.toml
+++ nethsm-sdk-rs/Cargo.toml
@@ -44,7 +44,7 @@ name = "basic"
 path = "tests/basic.rs"
 
 [dependencies.base64]
-version = "0.21"
+version = "0.22"
 features = ["alloc"]
 default-features = false
 
@@ -87,7 +87,7 @@ version = "0.4.38"
 [dev-dependencies.env_logger]
 version = "0.11.3"
 
-[dev-dependencies.rustainers]
+[disabled.dev-dependencies.rustainers]
 version = "0.12.0"
 
 [dev-dependencies.tokio]
